### PR TITLE
Option in Notifications to turn ON/OFF the LED Speaker CPU Monitor Animation

### PR DESCRIPTION
**Pull request by radujipa**

### CODE REVIEW

**Comment by radujipa** [comment]
https://github.com/KanoComputing/peldins/issues/2146

File: kano_settings/set_notifications.py [hunk 1]
```diff
@@ -36,10 +41,19 @@ def __init__(self, win):
         self.enable_all_radiobutton = self.get_button(0)
         self.disable_all_radiobutton = self.get_button(1)
         self.disable_world_radiobutton = self.get_button(2)
-        self.show_configuration()
+
+        if is_model_2_b() is True:
```
Comment (convolu) [inline kano_settings/set_notifications.py]: `is True` is redundant, I think

Comment (radujipa) [inline kano_settings/set_notifications.py]: Good spot, thank you!

**Comment by radujipa** [comment]
I've changed the behaviour of the checkbox slightly here.

 - rpi1: there is no checkbox, feature available for rp2 and upwards (OFF by default)
 - rpi2: checkbox only shows when the LED Speaker is plugged in (ON by default)

**Comment by tombettany** [comment]
:+1:

**Pull request merged**